scripts/ci/common.sh

```diff
@@ -17,7 +17,10 @@ function runTests() {
 }
 
 if [ -n "${APM_SERVER_BRANCH}" ]; then
- BUILD_OPTS="${BUILD_OPTS} --apm-server-build https://github.com/elastic/apm-server.git@${APM_SERVER_BRANCH}"
+ APM_SERVER_BRANCH_VERSION=${APM_SERVER_BRANCH%;*}
+ APM_SERVER_BRANCH_TYPE=${APM_SERVER_BRANCH//$APM_SERVER_BRANCH_VERSION/}
+ APM_SERVER_BRANCH_TYPE=${APM_SERVER_BRANCH_TYPE//;/}
+ BUILD_OPTS="${BUILD_OPTS} --apm-server-build https://github.com/elastic/apm-server.git@${APM_SERVER_BRANCH_VERSION} ${APM_SERVER_BRANCH_TYPE} "
```

Comment:
I like this idea for master but since 6.x will be mostly unchanging soon wouldn't it make sense to continue using the nightly snapshots instead?

Reply:
if the parameter is  "VERSION" and it's not "VERSION;--release" it would use the snapshot, for the server we use the snapshot for 6.5 right now.